packages/react-storage/src/components/StorageBrowser/views/LocationActionView/DeleteFilesControls.tsx

```diff
@@ -40,22 +41,21 @@ export const DeleteFilesControls = ({
   });
 
   const contextValue: ControlsContext = {
-    data: { taskCounts, tableData },
+    data: {
+      taskCounts,
+      tableData,
+      isActionStartDisabled: disablePrimary,
+      actionStartLabel: 'Start',
+    },
     actionsConfig: { type: 'BATCH_ACTION', isCancelable: true },
```

Comment:
```suggestion
```
nit:
Same suggestion as above on removing actionsConfig if it's not required.
Note: not sure if it's used by the `DataTableControl` or other components

Reply:
Yeah, I didn't add it so don't want to remove it as it might be used anywhere, same for the other similar comment.